### ISSUE TITLE
attachments w/ isAudioMessage:true not playable on iOS

### ISSUE THREAD

**Issue by d4tocchini** [BlueBubblesApp/bluebubbles-server#674]
When I send an attachment & set isAudioMessage true, the message works correctly on recipient's MacOS Messages.app:

<img width="333" alt="image" src="https://github.com/user-attachments/assets/5ba69f06-0d8c-433a-a6ef-50bbe7244217">

But, the same message on iOS shows up like below and is not clickable or playable:

<img width="571" alt="Screenshot 2024-07-17 at 9 36 00 PM" src="https://github.com/user-attachments/assets/eeb27d4a-214a-47cd-b66b-d091462d6e33">

To make it playable I have to turn off the isAudioMessage option. This makes the audio showup the same as above but at least its playable on iOS.

The audio buffer is an mp3 and whether I use isAudioMessage or not, it seems to always convert to caf. Is this expected?

**Comment by zlshames** [comment]
Yes, when you take an audio note in iMessage, it sends as a .caf, which is why we convert it when isAudioMessage is true. 

If you send an audio message with isAudioMessage being disabled, how does it show on the Mac? On the iPhone you said it shows correctly, but what about the mac

**Comment by rmajidy** [comment]
@zlshames did anyone ever figure this out? same thing happening to us @d4tocchini
